new/components/utils

```diff
@@ -385,6 +385,17 @@ function create_service_user() {
         ctx logger info "Creating user ${user} with home ${home}..."
         sudo useradd --shell /sbin/nologin --home-dir "${home}" --no-create-home --system "${user}"
     fi
+
+function clean_var_log_dir() {
+    ###
+    # Cleans up unused /var/log directory for named application.
+    # Directory must be empty or this will fail.
+    ###
+    for log in $(find /var/log/${1} -type f); do
+        sudo mv ${log} /var/log/cloudify/${1}/${log##/var/log/${1}/}-from_bootstrap
+    done
+    ctx logger info "Removing unnecessary logs directory: /var/log/${1}"
+    sudo rmdir /var/log/${1}
```

Comment:
It's not, it'll fail if the dir doesn't exist (so on second execution).

I'll make a couple of changes and upload them shortly.